.local/bin/statusbar/sb-bluetooth

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# Shows names of paired and connected bluetooth devices if bluetooth is on
+
+case $BLOCK_BUTTON in
+	1) dmenubluetooth; pkill -RTMIN+2 dwmblocks ;;
+	3) notify-send " Bluetooth module" "\- Shows paired devices.
+- Click to configure bluetooth." ;;
+	6) "$TERMINAL" -e "$EDITOR" "$0" ;;
+esac
+
+if bluetoothctl show | grep -q 'Powered: yes'; then
```

Comment:
If the bluetoothd service isn't started when opening dwm this code will prevent dwmblocks from opening since it's waiting for a response from `bluetoothctl show` which it will never get so there should be a timeout to prevent that.
```suggestion
if timeout 0.1s bluetoothctl show | grep -q 'Powered: yes'; then
```